### ISSUE TITLE
Should BASE have a Preferences Dialog?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#513]
@jbphet questioned this during a discussion today. The Preferences Dialog in BASE will only have one control - "All Audio", which is the same control that exists in the Navigation Bar. As such, there may not be much benefit in including it until more features are included. @emily-phet can you please comment on whether you would still like to have this Dialog in BASE for the upcoming publication with sounds?

**Comment by jbphet** [comment]
I'd like to add that during a discussion yesterday, @emily-phet mentioned that John Travoltage is also going to be published soon.  If the goal of putting the preferences dialog into BASE was to get some feedback on it, perhaps we can get it through JT instead.

I think we should consider this a general decision about whether the preferences dialog should be present in sims that have nothing other than non-enhanced sounds for this dialog to manage.  In this case, it essentially reduces to a redundant way to do the same thing, i.e. turn sounds on and off, which would certainly seem odd to me if I were a user.

**Comment by emily-phet** [comment]
@jessegreenberg @terracoda and I met today to discuss this and a few other issues that have come up re the preferences menu. 

We all heartily agreed that from a local interaction design perspective, having the preferences menu contain only one actionable preference that is essentially redundant with a button in the nav bar is clunky.

From a global perspective across the project, having a point in the near future where there is universal use of the Preferences menu going forward has some benefits:
- Consistent look and feel of the nav bar 
- Consistency in teacher professional development materials, workshops, third party company, and funder conversations in the appearance and overview of PhET, sim interfaces, etc. 
- The opportunity to introduce the wide range of features PhET offers (i.e., through the language in the general tab - which is being revised based on recent feedback, and through professional development materials, workshops, etc.), even when demoing sims without some or any of these features. 
- Avoiding scenarios where there might be some confusion around regarding what features are "Preferences" (e.g., if some features require a preferences menu but others don't, some within and outside the team may get confused about what we mean by "Preference"). 

I'd like to ask @kathy-phet her thoughts on this, and have made plans to speak with her later this afternoon. I'll update this issue with the outcome from that discussion.

**Comment by emily-phet** [comment]
Two updates:

Conversation with @kathy-phet:
- She was still in favor of taking a uniform approach to the implementation of the Preferences menu, the approach we had leaned towards from the early group design iterations, even if some sims have a very minimal Preferences menu. 

Addition of Interactive Highlights:
- I checked with @jessegreenberg about potentially adding interactive highlights to BASE. These are pink focus highlights automatically displayed when using focus-based alternative input (e.g., keyboard navigation), but when "Interactive Highlights" is enabled in the Visual Display preferences, the pink focus highlights are displayed regardless of input type. So, for example, this is a way a user could opt in to having the focus highlights when using mouse or touch. 
@jessegreenberg was able to implement this quickly, and now BASE Preferences menu has three tabs: General, Visual, and Audio, with a total of two preferences they can change, Sound on/off and Interactive Highlights on/off. 

Assuming all goes well with interactive highlights moving through QA, I think this addresses the issue of BASE having too minimal of a Preferences menu. It also may indicate that along with keyboard navigation, interactive highlights is a really light lift to incorporate into sims, and could become standard going forward, further populating future Preferences menus.
